packages/shylu/shylu_dd/frosch/src/Tools/FROSch_Tools_decl.hpp

```diff
@@ -48,6 +48,9 @@
 #include <Xpetra_MultiVectorFactory.hpp>
 #include <Xpetra_VectorFactory.hpp>
 #include <Xpetra_ExportFactory.hpp>
+#include <Zoltan2_MatrixAdapter.hpp>
+#include <Zoltan2_XpetraCrsMatrixAdapter.hpp>
+#include <Zoltan2_PartitioningProblem.hpp>
```

Comment:
We need an optional dependency on Zoltan2 as an optional dependency. Then, we have to add some `#ifdef`